readthedocs/projects/models.py

```diff
@@ -2015,6 +2015,10 @@ def add_features(sender, **kwargs):
                 "Proxito: Use new unresolver implementation for serving documentation files."
             ),
         ),
+        (
+            USE_OLD_PROXITO_IMPLEMENTATION,
+            _("Proxito: Use old implementation for serving documentation files."),
```

Comment:
```suggestion
            _("Proxito: Use old Proxito implementation (no unresolver) for serving documentation files."),
```

Make this sentence more explicit here.